packages/containers/src/ObjectViewer/ObjectViewer.container.js

```diff
@@ -95,6 +96,9 @@ class ObjectViewer extends React.Component {
 
 	render() {
 		const state = (this.props.state || DEFAULT_STATE).toJS();
+		if (this.props.state === DEFAULT_STATE && this.props.openFirst) {
```

Comment:
Did you mean `(state === DEFAULT_STATE && this.props.openFirst)` ?

Reply:
no beacuse state is not immutable it's the result of toJS()

Reply:
here this is a kind of pristine test